modules/candidate_parameters/jsx/CandidateInfo.js

```diff
@@ -64,6 +64,34 @@ var CandidateInfo = React.createClass(
         formData: formData
       });
     },
+
+    addListItem: function(formElement, value, pendingValKey) {
```

Comment:
what should happen once you add a favourite snack? when i update the form with the tag element, and refresh, the tag elements disappear

Reply:
TagsElements have arrays held in the formData state object that gets added to with each item. they won't persist after a refresh since i did not do anything on the backend for them